matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/crypto/tasks/UploadSignaturesTask.kt

```diff
@@ -37,7 +37,9 @@ internal class DefaultUploadSignaturesTask @Inject constructor(
     override suspend fun execute(params: UploadSignaturesTask.Params) {
         try {
             val response = executeRequest<SignatureUploadResponse>(eventBus) {
-                apiCall = cryptoApi.uploadSignatures(params.signatures)
+                this.isRetryable = true
```

Comment:
Why adding this?

Reply:
I guess you wanted to retry the uploadSignatureTask if you have no network connectivity?
We can't wait for network anymore as it's not reliable...